pkg/operator/ceph/controller/cluster_info.go

```diff
@@ -285,7 +286,7 @@ func loadMonConfig(clientset kubernetes.Interface, namespace string) (map[string
 
 	// Parse the monitor List
 	if info, ok := cm.Data[EndpointDataKey]; ok {
-		monEndpointMap = ParseMonEndpoints(info)
+		monEndpointMap = endpoint.ParseMonEndpoints(info)
```

Comment:
I was thinking if we can just call 

mon.ParseMonEndpoints(info),

rom where,

Maybe other people can also make sure is adding a new package make sense

Reply:
@subhamkrai @sp98 @BlaineEXE